.github/workflows/build-docker-images.yml

```
@@ -0,0 +1,28 @@
+# This workflow is used to build docker images
+
+name: Docker Image CI
```

```suggestion
name: Docker Image CI for ppid-retrieval-tool
```
nit: suggest to have a more specific name